xla/xla_data.proto

```diff
@@ -933,7 +933,18 @@ message PrecisionConfig {
 
     // Next: 4
   }
-  repeated Precision operand_precision = 1;
+  enum Gradient {
```

Comment:
Gradient is a misleading name, because a Gradient can be an Activation or a Gradient. Can you please add documentation on the intended use?